hail/src/main/scala/is/hail/expr/ir/EmitStream.scala

```diff
@@ -50,12 +68,65 @@ object COption {
   def apply[A](missing: Code[Boolean], value: A): COption[A] = new COption[A] {
     def apply(none: Code[Ctrl], some: A => Code[Ctrl])(implicit ctx: EmitStreamContext): Code[Ctrl] = missing.mux(none, some(value))
   }
+
+  // Empty is the only COption allowed to not call `some` at compile time
```

Comment:
Should `"Empty"` in this comment actually be `"None"`?

Reply:
Oops, yes.